File: app/src/main/java/com/v2/desafionubank/model/Links.java
package com.v2.desafionubank.model;

import com.google.gson.annotations.SerializedName;

/**
 * Created by CaioSChristino on 10/04/18.
 */

public class Links {
    private LinkHref notice;
    private LinkHref self;
    private LinkHref chargeback;
    @SerializedName("block_card")
    private LinkHref blockCard;
    @SerializedName("unblock_card")
    private LinkHref unblockCard;

    public LinkHref getNotice() {
        return notice;
    }

    public LinkHref getSelf() {
        return self;
    }

    public LinkHref getBlockCard() {
        return blockCard;
    }

    public LinkHref getUnblockCard() {
        return unblockCard;
    }

    public LinkHref getChargeback() {
        return chargeback;
    }
}

File: app/src/main/java/com/v2/desafionubank/model/Links.kt
package com.v2.desafionubank.model

import com.google.gson.annotations.SerializedName


/**
 * Created by CaioSChristino on 18/04/18.
 */

class Links {
    val notice: LinkHref? = null
    val self: LinkHref? = null
    val chargeback: LinkHref? = null
    @SerializedName("block_card")
    val blockCard: LinkHref? = null
    @SerializedName("unblock_card")
    val unblockCard: LinkHref? = null
}